client/src/auth/keycloakHelpers.ts

```diff
@@ -0,0 +1,52 @@
+import { KeycloakInitOptions } from "keycloak-js";
+
+// get tokens from localstorage
+export const getLocalTokens = () => {
+  const token = localStorage.getItem('token');
+  const refreshToken = localStorage.getItem('refreshToken');
+  return [token, refreshToken];
+};
+
+// save tokens to localstorage
+export const setLocalTokens = ({ token, refreshToken } : { token: string | undefined, refreshToken: string | undefined}) => {
```

Comment:
Out of curiosity - why we doing this? This is security risk as localstorage is not protected by domain same as cookies.

Reply:
Keycloak keeps tokens for this very reason in cookies.

Reply:
Keycloak on init should rely on cookie anyway